app/features/TransactionSlice.ts

```diff
@@ -216,13 +212,12 @@ export const loadTransactions = createAsyncThunk(
             }
         };
 
-        const minId = state.transactions.transactions
+        // The array of transactions is sorted in descending order on the id. Therefore the
+        // minimum id can be found as the final transaction in the array.
+        const transactionIdArray = state.transactions.transactions
```

Comment:
Here, couldn't we assume that if the last element didn't have an id, then none of them did?

Reply:
We could but it would be more fragile to future changes.